scenes/menus/title/components/main_menu.gd

```diff
@@ -16,6 +18,10 @@ func _ready() -> void:
 	# custom code in the HTML shell that we do not have yet.
 	quit_button.visible = OS.get_name() != "Web"
 
+	continue_button.visible = GameState.can_restore()
+	if GameState.can_restore():
+		start_button.text = "Restart"
```

Comment:
It will be a bit hard to test the initial no-saved-state state because the only in-game way to clear the saved state is to start a new game, and entering Fray's End saves state. But there is a way:

- Restart
- Quit during intro

or just delete the file.

Reply:
That's exactly how I tested it. Sorry, I forgot to mention it.